SOURCE_OFFER

```diff
@@ -0,0 +1,19 @@
+Freedom of the Press Foundation (FPF) maintains custom-compiled Linux
+kernel images for use in SecureDrop. The kernel source has been patched
+with the grsecurity modifications, and only binaries are available in the
+FPF apt repository.
+
+Under the terms of the GPL, any user with access to the binaries has a right
```

Comment:
s/with access to the binaries/in possession of this offer/

I'd be happy to explain the rationale, if you'd like :-)